templates/src/index.js

```diff
@@ -0,0 +1,25 @@
+import { getOptions } from 'loader-utils';
+import validateOptions from 'schema-utils';
+import schema from './options.json';
+
+export const raw = true;
```

Comment:
Maybe we should add a comment to the Loader API (`https://webpack.js.org/api/loaders/#-raw-loader`) with a short intro/instruction here and comment that line out by default ?

We could do the same analog for `export function pitch () {}` (Pitching Loader) to complete the `loader` template

```js
/**
 * Raw Loader
 * ...
 */
// export const raw = true;
/**
 * Pitching Loader
 * ...
 */
// export function pitch (request) { ... };
/**
 * Loader (Default)
 * ...
 */
export default function loader (src, map, meta) { ... };

Reply:
See comment below about `index.js`